tests/system/test_gbq.py

```diff
@@ -924,6 +924,28 @@ def test_upload_data(self, project_id):
         )
         assert result["num_rows"][0] == test_size
 
+    def test_upload_empty_data(self, project_id):
+        test_id = "data_with_0_rows"
+        test_size = 0
+        df = DataFrame()
```

Comment:
Looks like we might have an additional problem when the DataFrame contains no columns.

In the conda build (https://circleci.com/gh/tswast/pandas-gbq/276) I'm getting:

```
E           google.api_core.exceptions.BadRequest: 400 POST https://www.googleapis.com/upload/bigquery/v2/projects/pandas-gbq-tests/jobs?uploadType=resumable: Empty schema specified for the load job. Please specify a schema that describes the data being loaded.
```

Since we still create a table in pandas-gbq before running the load job, we can probably avoid doing the load job altogether when a DataFrame contains no rows.